SCClassLibrary/Common/GUI/PlusGUI/Control/NodeTreeView.sc

```diff
@@ -8,20 +8,28 @@ NodeTreeView {
 		^super.newCopyArgs(server).makeWindow(bounds, parent)
 	}
 
-	makeWindow { |bounds, parent|
+	makeWindow 	{ |bounds, parent|
+		var scrollView;
```

Comment:
Can you move it so that it's together with other instance variables? Technically it doesn't have to be, but we're not using local variables elsewhere in this method so I'd like to do that for consistency.

It can go after `view` or be in a separate line.

Reply:
Done in https://github.com/supercollider/supercollider/pull/6948/commits/5a271ff11407fb63c63ecf89bab8c9a3e0901415!